app/views/submissions/index.html.erb

```diff
@@ -2,141 +2,153 @@
 
 <% content_for :stylesheets do %>
   <%= stylesheet_link_tag "datatable.adapter" %>
-
-  <style>
-    table [type="checkbox"] + label {
-      width: 18px;
-      padding: 0;
-    }
-
-    #floater {
-      z-index: 5;
-    }
-  </style>
+  <%= stylesheet_link_tag "datatables-rows" %>
 <% end %>
 
 <% content_for :javascripts do %>
   <%= external_javascript_include_tag "lodash" %>
   <%= javascript_include_tag "sorttable" %> 
   <%= external_javascript_include_tag "jquery.dataTables" %>
+  <%= external_javascript_include_tag "datatables-buttons" %>
   <%= javascript_include_tag "manage_submissions" %>
-
+  <%= javascript_include_tag "datatables-rows" %>
 <% end %>
+
 <h3>Manage Submissions</h3>
 <hr>
 
-<p>
-    <%= link_to "Create New Submission".html_safe, new_course_assessment_submission_path(@course, @assessment),
-        {:title=>"Create a new submission for a student, with an option to submit a handin file on their behalf",
-         :class=>""} %>
-         |
-    <%= link_to "Download All Submissions".html_safe,
-        downloadAll_course_assessment_submissions_path(@course, @assessment),
-        {:title=>"Down all submissions from each student",
-         :class=>""} %>
-         |
-    <%= link_to "Download Final Submissions".html_safe,
-        downloadAll_course_assessment_submissions_path(@course, @assessment, final: true),
-        {:title=>"Download the most recent submission from each student",
-         :class=>""} %>
-         |
+<!--
```

Comment:
may I ask why is this commented out? If it's not needed I think it's best to remove it

Reply:
Will remove!